tests/integration-tests/src/flows/faucet/initFaucet.ts

```diff
@@ -0,0 +1,45 @@
+import { FlowProps } from '../../Flow'
+import { extendDebug } from '../../Debugger'
+import { FixtureRunner } from '../../Fixture'
+import { BuyMembershipHappyCaseFixture, TransferInvitesHappyCaseFixture } from '../../fixtures/membership'
+import { SetBudgetFixture } from '../../fixtures/workingGroups/SetBudgetFixture'
+import _ from 'lodash'
+import BN from 'bn.js'
+
+export default async function initFaucet({ api, env, query }: FlowProps): Promise<void> {
+  const debug = extendDebug('flow:initFaucet')
+  debug('Started')
+
+  // Get membership working group leader
+  const membershipWorkingGroup = 'membershipWorkingGroup'
+  const [, membershipLeader] = await api.getLeader(membershipWorkingGroup)
+
+  // Grant lead invitation quota
+  const proposalDetails = api.createType('ProposalDetails', { SetMembershipLeadInvitationQuota: 50 })
```

Comment:
This isn't doing anything.

Instead use extrinsic on the membership module `setLeaderInvitationQuota` with sudo to grant them invites. If there is no existing fixture then we should add it.

Reply:
Done